plugins/packages/common/lib/query.error.ts

```diff
@@ -1,11 +1,13 @@
 export class QueryError extends Error {
   data: Record<string, unknown>;
   description: any;
-  constructor(message: string | undefined, description: any, data: Record<string, unknown>) {
+  metadata?: any;
```

Comment:
Prefer `unknown` over `any`. Let's change the description to `string`, if there are type issues then make it `unknown`.